completion/waffle.py

```diff
@@ -27,3 +19,17 @@ def waffle():
     Returns the namespaced, cached, audited Waffle class for completion.
     """
     return WaffleSwitchNamespace(name=WAFFLE_NAMESPACE, log_prefix='completion: ')
+
+
+def waffle_switch(name):
+    """
+    Return the waffle switch associated to the completion namespace.
+    """
+    # .. toggle_name: completion.enable_completion_tracking
+    # .. toggle_implementation: WaffleSwitch
+    # .. toggle_default: False
+    # .. toggle_description: Indicates whether or not to track completion of individual blocks. Keeping this disabled
+    #   will prevent creation of BlockCompletion objects in the database, as well as preventing completion-related
+    #   network access by certain xblocks.
+    # .. toggle_use_cases: open_edx
+    return WaffleSwitch(waffle(), name, module_name=__name__)
```

Comment:
It doesn't make sense to me to have a general function, but annotations for a single switch.
Could this instead be?:
```
# .. toggle_name: completion.enable_completion_tracking
# .. toggle_implementation: WaffleSwitch
# .. toggle_default: False
# .. toggle_description: Indicates whether or not to track completion of individual blocks. Keeping this disabled
#     will prevent creation of BlockCompletion objects in the database, as well as preventing completion-related
#     network access by certain xblocks.
# .. toggle_use_cases: open_edx
ENABLE_COMPLETION_TRACKING_SWITCH = WaffleSwitch(waffle(), ENABLE_COMPLETION_TRACKING, module_name=__name__)
```

Reply:
Yes, this makes much more sense.